atem_dissector.lua

```diff
@@ -98,6 +98,7 @@ pf_fields["pf_field_unknown0"]  = ProtoField.new  ("Unknown", "atem.field.unknow
 VALS["VALS__TOP_HASSDOUTPUT"] = {[0] = "No", [1] = "Yes"}
 pf_fields["pf_cmd__top_hassdoutput"]    = ProtoField.new  ("Has SD Output", "atem.cmd._top.hassdoutput", ftypes.UINT8, VALS["VALS__TOP_HASSDOUTPUT"], base.DEC)
 
+pf_fields["pf_field_padding"]  = ProtoField.new  ("Padding", "atem.field.padding", ftypes.NONE, nil, base.NONE)
```

Comment:
Hi Julian, I worked on this a while ago; but given that those bytes appear to be random, don't seem to have an effect on the ATEM and always round commands to a multiple of 4, we have enough information to infer that they are padding bytes.
If you don't think that's enough, it can of course be changed to `unknown`